src/js/components/Select/index.d.ts

```diff
@@ -56,7 +71,13 @@ export interface SelectProps extends BasicSelectProps {
   defaultValue?: string | number | object | (string | number | object)[];
   multiple?: boolean;
   selected?: number | number[];
-  value?: string | JSX.Element | number | object | (string | number | object)[];
+  value?:
+    | boolean
```

Comment:
Why did we include `boolean` here? Looking at the docs it isn't listed there

Reply:
Made the required change.